i2c/24lc32_i2c/READ_ME.adoc

```diff
@@ -0,0 +1,30 @@
+= Reading and writing 24LC32 EEPROM via I2C
+
+This example code demonstrates how to read and write data from 24LC32 EEPROM. It contains a library with functions for reading data, writing single bytes, and writing pages, and an example which uses these functions.
```

Comment:
```suggestion
This example code demonstrates how to read and write data from 24LC32 EEPROM. It contains a library with functions for reading data, writing single bytes, and writing pages; and an example which uses these functions.
```
(otherwise it looks a bit weird with two "and"s :slightly_smiling_face: )